bindings/api.go

```diff
@@ -63,6 +63,15 @@ func GetFundStatus(notificationToken string) ([]byte, error) {
 	return marshalResponse(breez.GetFundStatus(notificationToken))
 }
 
+/*
+SendSwapInvoice is part of the binding inteface which is delegated to breez.SendSwapInvoice
+*/
+func SendSwapInvoice(sendSwapInvoiceRequest []byte) error {
```

Comment:
Why SendSwapInvoice is exported?
I was thinking of the flow as automatic, means:
1. LND watch for the swap address
2. Once the amount is confirmed, generate invoice and call GetPayment